packages/components/src/index.html

```diff
@@ -26,12 +26,17 @@
 
       window.courierAsyncInit = window.courierAsyncInit || [];
       window.courierAsyncInit.push(() => {
-        window.showToastOnClick = () => {
-          window.courierSdk.toast({
-            title: "Hello",
-            body: "World",
-          });
-        };
+        window.courierSdk.on("toast/init", () => {
+          window.showToastOnClick = () => {
+            window.courierSdk.toast({
+              title: "Hello",
+              body: "World",
+            });
+          };
+
+          const showToast = document.getElementById("show-toast");
+          showToast.removeAttribute("disabled");
```

Comment:
What is the purpose of disabled to being with?

Reply:
just so i can see the button turn on and off and that things are working.  for development only